io.github.torrent_file_editor.Torrent-file-editor.yaml

```
@@ -0,0 +1,41 @@
+app-id: io.github.torrent_file_editor.torrent-file-editor
+runtime: org.kde.Platform
+runtime-version: "5.15-21.08"
+sdk: org.kde.Sdk
+
+rename-desktop-file: torrent-file-editor.desktop
+rename-appdata-file: torrent-file-editor.appdata.xml
+#rename-icon: torrent-file-editor.png
```

```suggestion
rename-icon: torrent-file-editor
```

you likely need this.

---

thank you very much I was having a bad time with that line